tests/fixtures/network/node_nics.py

```
@@ -0,0 +1,166 @@
+    Function removes ipv4 from secondary nics.
+    """
+    for worker_node in schedulable_nodes:
+        worker_nics = nodes_available_nics[worker_node.name]
+        with EthernetNetworkConfigurationPolicy(
+            name=f"disable-ipv4-{name_prefix(worker_node.name)}",
+            client=admin_client,
+            node_selector=get_node_selector_dict(node_selector=worker_node.hostname),
+            interfaces_name=worker_nics,
+        ):
+            LOGGER.info(
+                f"selected worker node - {worker_node.name} under NNCP selected NIC information - {worker_nics} "
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)